### PR TITLE
Request to add our xkode.io dapp to Nova wallet

### PR DESCRIPTION
**Pull request by XHYPEToken**
Hello, we are the Xhype team at https://xhype.io and we have been in contact with you for a time.

We are finishing integrating our dapp into Nova Wallet and we ask that you please include it in your catalog.

We have carried out all the steps for this.

Thank you.

### CODE REVIEW

**Comment by XHYPEToken** [comment]
Good morning, any reviewer for this pull request please?

File: dapps/dapps_dev.json [hunk 1]
```diff
@@ -966,5 +966,13 @@
         "test"
       ]
     }
+    {
+      "name": "Xkode",
+      "url": "https://xkode.io",
+      "icon": "https://raw.githubusercontent.com/novasamatech/nova-utils/master/icons/dapps/color/xhype.png",
+      "categories": [
+        "utilities"
```
Comment (leohar) [inline dapps/dapps_dev.json:974]:
```suggestion
        "utilities",
        "test"
```

**Comment by leohar** [comment]
> Good morning, any reviewer for this pull request please?
Thank you for your PR, Nova doesn't support BNB Smartchain atm, we will merge it in dev and promote after app will support   network

**Review by leohar — APPROVED** [review]

**Comment by XHYPEToken** [comment]
> > Good morning, any reviewer for this pull request please?
> > Thank you for your PR, Nova doesn't support BNB Smartchain atm, we will merge it in dev and promote after app will support   network

Helo leohar, thanks for your help, I think there has been some misunderstanding, our marketplace supports purchases on almost all networks, including Polkadot, which is why we are integrating into Nova Wallet:

https://xkode.io

You already can log in with NovaWallet and purchase products in polkadot using xpay payment gateway.

Thanks

**Pull request merged**